packages/jaeger-ui/src/components/common/CopyIcon.tsx

```diff
@@ -66,14 +67,11 @@ export default class CopyIcon extends React.PureComponent<PropsType, StateType>
         mouseLeaveDelay={0.1}
         onVisibleChange={this.handleTooltipVisibilityChange}
         placement={this.props.placement}
-        title={this.state.hasCopied ? 'Copied' : this.props.tooltipTitle}
+        title={this.state.hasCopied ? 'Copied' : ''}
```

Comment:
will the empty string make the tooltip not show and only appear on copy?

Reply:
yes